Home_work10/Home_work10.cpp

```diff
@@ -0,0 +1,84 @@
+﻿#include <iostream>
+
+enum class SortingDirection {
+	ascending,
+	descending
+};
+
+bool isSorted(const int* arr, SortingDirection direction) {
+	if (int(direction) == int(SortingDirection::descending)) {
+		for (int i = 1; i <= 10; ++i) {
```

Comment:
you should pass size as parameter. What if sise of input array is different than 10?